metadata-ingestion/src/datahub/specific/ownership.py

```diff
@@ -0,0 +1,48 @@
+from typing import Generic, List, Optional, TypeVar
+
+from datahub.emitter.mcp_patch_builder import MetadataPatchProposal
+from datahub.metadata.schema_classes import (
+    OwnerClass,
+    OwnershipClass,
+    OwnershipTypeClass,
+)
+
+T = TypeVar("T", bound=MetadataPatchProposal)
+
+
+class OwnershipPatchHelper(Generic[T]):
```

Comment:
I think this one probably should be a mixin class or something - the composition interface is a bit weird

Reply:
leaving this in for now, can get to this later.